rules/windows/execution_via_xp_cmdshell_mssql_stored_procedure.toml

```
@@ -16,10 +16,59 @@ index = ["winlogbeat-*", "logs-endpoint.events.*", "logs-windows.*"]
 language = "eql"
 license = "Elastic License v2"
 name = "Execution via MSSQL xp_cmdshell Stored Procedure"
-note = """## Setup
+note = """## Triage and analysis
+
+### Investigating Execution via MSSQL xp_cmdshell Stored Procedure
+
+Microsoft SQL Server (MSSQL) has procedures meant to extend its functionality, the Extended Stored Procedures. These
+procedures are external functions written in C/C++; some provide interfaces for external programs. This is the case for
+xp_cmdshell, which spawns a Windows command shell and passes in a string for execution, meaning that attackers can use
+it to execute commands on the system running the SQL server, commonly to escalate their privileges and establish
+persistence.
```

Breaking up a long sentence:
```suggestion
procedures are external functions written in C/C++; some provide interfaces for external programs. This is the case for
xp_cmdshell, which spawns a Windows command shell and passes in a string for execution. Attackers can use this to execute commands on the system running the SQL server, commonly to escalate their privileges and establish
persistence.
```